docs/README.md

```diff
@@ -4,9 +4,10 @@ Get inferencing running on Kubernetes: LLMs, Embeddings, Speech-to-Text.
 
 ✅️  Drop-in replacement for OpenAI with API compatibility  
 🧠  Serve top OSS models (LLMs, Whisper, etc.)  
-🚀  Multi-platform: CPU-only, GPU, coming soon: TPU  
+🚀  Multi-platform: CPU-only, GPU, TPU  
+💾  Model caching with shared filesystems (EFS, Filestore, etc.) -> From 10 min to 2 min to load 70B model  
```

Comment:
`-> From 10 min to 2 min to load 70B model` throws off the symmetry of all the other section lengths. I would perhaps advertise this elsewhere.

Reply:
it's too long, maybe I can make it go to the next line?

Reply:
I think perf numbers warrant their own section